spec.bs

```diff
@@ -763,70 +763,258 @@ enrollment and authentication. This extends the
-        payments-focused browser UX during SPC enrollment.
+* When the attacker initiates SPC, the user will be shown an interface by the
+    User Agent that clearly states the transaction details (including the payee
+    and amount). The user is very likely to reject this interface as invalid.
+* If the user does agree to the transaction, and completes the subsequent
+    WebAuthn authentication ceremony, the attacker now has a signed SPC
+    assertion for the [=Relying Party=].
+* However, the [=Relying Party=] is not expecting a transaction, and so will
+    just reject the assertion it was sent!
 
-    * In the absence of such as UX, a browser can require a
-        user gesture as part of initiating the API.
+NOTE: In addition the above, the assertion `challenge` will also be incorrect.
 
-# Privacy Considerations # {#sctn-privacy-considerations}
+The general concept here is that a Secure Payment Confirmation assertion is
+essentially useless unless it is part of an ongoing online transaction.
 
-Note: This section is still very much a draft.
+## Merchant-supplied authentication data ## {#sctn-security-merchant-data}
 
-SPC primarily points to WebAuthn for its privacy model, but it does relax
-that model in significant ways that should be considered.
+A consequence of this specification's third-party authentication ceremony is
+that even in a valid transaction (i.e. one that the [=Relying Party=] is
+expecting), a third-party provides the transaction details that are shown to
+the user:
 
-* Probing for credential ids
+* Transaction amount and currency
+* Payment instrument name and icon
+* Payee origin
 
-    * Currently SPC exits immediately (on Chrome) when no passed credential
-        matches the current device. This could be used to probe for whether the
-        current device matches an existing known credential. This does show a
-        very clear payments UX when a match is found (though that may be too late).
+This could lead to a spoofing attack, in which a merchant presents incorrect
+data to the user. For example, the merchant could tell the bank (in the
+backend) that it is initiating a purchase of $100, but then pass $1 to the SPC
+API (and thus show the user a $1 transaction to verify).
 
-        Note: The instant-exit may be removed in the future.
+Secure Payment Confirmation actually makes defeating this kind of attack easier
+than it currently is on the web. In online payments today, the bank has to
+trust that the merchant showed the user the correct amount in their checkout
+flow (and any fraud discoveries are post-payment, when the user checks their
+account statement).
 
-* Identifying user across different payment instruments.
+With Secure Payment Confirmation, the bank can (and should) instead
+[[#sctn-verifying-assertion|verify the cryptogram]] that they receive, to
+ensure that it aligns with the transaction details that the merchant initiated
+the transaction for.
```

Comment:
Editorial: "that it aligns with the transaction details provided by the merchant."